Notifications/NotificationService.swift

```diff
@@ -9,23 +9,23 @@
 import UserNotifications
 
 class NotificationService: UNNotificationServiceExtension {
-    
+
     var contentHandler: ((UNNotificationContent) -> Void)?
     var bestAttemptContent: UNMutableNotificationContent?
-    
+
     override func didReceive(_ request: UNNotificationRequest,
                              withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
-        
+
         func save(_ identifier: String, data: Data, options: [AnyHashable: Any]?) -> UNNotificationAttachment? {
-            
+
             var directory = URL(fileURLWithPath: NSTemporaryDirectory())
             directory = directory.appendingPathComponent(
                 ProcessInfo.processInfo.globallyUniqueString,
                 isDirectory: true)
-            
+
             do {
                 try FileManager.default.createDirectory(at: directory, withIntermediateDirectories: true, attributes: nil)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/adb6600e04f8d78e85d3ed821c2844be0fbd4321/Notifications/NotificationService.swift#L27"></span>Line should be 120 characters or less: currently 122 characters<br />
<code>line_length</code> <code>NotificationService.swift:27</code></td>
    </tr>
  </tbody>
</table>